lib/locomotive/steam/services/site_finder_service.rb

```diff
@@ -6,7 +6,7 @@ class SiteFinderService
       attr_accessor_initialize :repository, :request
 
       def find
-        repository.by_domain(request.host)
+        repository.by_domain(request.host) if request
```

Comment:
Sounds good. Takes care of that little code smell introduced by that conditional.